src/Runner.Listener/Program.cs

```diff
@@ -16,7 +16,7 @@ public static class Program
         public static int Main(string[] args)
         {
             // Add environment variables from .env file
-            LoadAndSetEnv();
+            EnvironmentParser.LoadAndSetEnvironment();
```

Comment:
should we get the return dictionary and set the environment variables there? otherwise, it's a little bit weird that the return value is only used by L0 tests.